src/main/java/codeu/controller/NotificationServlet.java

```diff
@@ -0,0 +1,47 @@
+package codeu.controller;
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
```

Comment:
I think some of these aren't used, let's scrub them to keep only what we need in this PR.